community-plugins.json

```diff
@@ -1281,13 +1281,21 @@
         "description": "An interface to experiment with python scripts and more.",
         "author": "Cristian Vasquez",
         "repo": "cristianvasquez/obsidian-lab"
+        "branch": "main"
```

Comment:
Looks like an issue with the rebase. Can you remove this? It should be on the `kanban` section instead.